### DISCUSSION TITLE
Is it possible to monitor flask socketio app using prometheus ?

### DISCUSSION THREAD

**Discussion by sparkingdark** [miguelgrinberg/python-socketio#1010] [Q&A]
Hello,
So basically I started looking into options so that I can monitor the python flask apps using Prometheus regarding there request and response time. Is it possible to do with the python flask socketio we are currently using this project into some of our projects.

For the HTTP service I used another open source library  [flask-prometheus](https://github.com/pilosus/flask_prometheus_metrics)
which can send the metrics to Prometheus, but this does not work for flask socketio.

If anyone used before anything related to monitoring let me know. 

Thanks in advance.

**Comment by miguelgrinberg** [comment]
You should be able to use any Socket.IO monitoring solution, doesn't need to be specific to Flask or Python.

**Reply by sparkingdark** [reply]
Okay thanks for the answer. Is there any way to know how socketio python creating connections like a more low level view.

**Reply by miguelgrinberg** [reply]
No, this library does not provide low-level access. But you can maintain any information that you need in your application and provide it to the monitoring agent.
